### ISSUE TITLE
Static phar release is still using outdated pdepend version

### ISSUE THREAD

**Issue by DanielSchwiperich** [phpmd/phpmd#569]
that the const visibility keyword was causing errors was fixed by updating the composer dependency for pdepend. See #436

The phar release on the website is still using pdepend 2.5.0 and was not updated

/cc @ravage84

**Comment by dbu** [comment]
is the static phar automatically created or is this a manual process? it seems to still use an outdated version of pdepend that complains about `private const`

**Comment by Anamelash** [comment]
ive downloaded v2.6.0 phar and got "Unexpected token: const" error

**Comment by goodevilgenius** [comment]
I also get the error "Unexpected token: |" when the code is like:

```php
try {
  doSomething();
} catch (RuntimeException | LogicException $e) {
  echo $e->getMessage();
}
```

That error doesn't occur when installed through composer.

**Comment by tvbeek** [comment]
@goodevilgenius Thanks for the feedback, I suspect there is a bug in the creation of the phar file.

**Comment by kylekatarnls** [comment]
Will be fixed in 2.8.0 release.
